statistics/queries/continent_histogram.sql

```diff
@@ -18,61 +19,60 @@ dl_per_location AS (
 ),
 # With good locations and valid IPs
 dl_per_location_cleaned AS (
-  SELECT
-    date,
-    continent_code,
-    mbps,
-    MinRTT,
-    ip
-  FROM dl_per_location
+  SELECT * FROM dl_per_location
   WHERE
     continent_code IS NOT NULL
     AND continent_code != ""
     AND ip IS NOT NULL
 ),
-# Gather descriptive statistics per geo, day, per ip
-dl_stats_perip_perday AS (
+
+# Gather rows per day and ip
+dl_rows_perip_perday AS (
   SELECT
     date,
     continent_code,
     ip,
-    MIN(mbps) AS download_MIN,
-    APPROX_QUANTILES(mbps, 100) [SAFE_ORDINAL(25)] AS download_Q25,
-    APPROX_QUANTILES(mbps, 100) [SAFE_ORDINAL(50)] AS download_MED,
-    AVG(mbps) AS download_AVG,
-    APPROX_QUANTILES(mbps, 100) [SAFE_ORDINAL(75)] AS download_Q75,
-    MAX(mbps) AS download_MAX,
-    APPROX_QUANTILES(MinRTT, 100) [SAFE_ORDINAL(50)] AS download_minRTT_MED
+    # This organizes all tests, in an arbitrary but repeatable order.
```

Comment:
@mattmathis suggests that this subquery should select one random row per IP/day/geo, and stats should be computed in a later subquery.

Reply:
Done in [2706553](https://github.com/m-lab/stats-pipeline/pull/42/commits/270655378d2897ab096b46d44e92ef7485f8848c)